gravityflow/gravityflowformconnector
# Add the form filter to more steps

re: [HS#12512](https://secure.helpscout.net/conversation/1066611510/12512?folderId=1113492)

This updates the New Entry and Update Fields steps to pass the local form through the `gravityflowformconnector_{$this->get_type()}_form` filter.

This will enable GP Populate Anything to hydrate the form before the mappings are processed.

## Testing Instructions
- Create a form with a checkbox field and text field.
- Use GP Populate Anything to populate the checkbox field choices. Ensure that whatever object you use to populate the field adds several choices.
- Add an Update Fields step to populate the text field with the selected checkbox choices from the current entry.
- With master preview the form and submit with all the checkbox choices selected.
- View the entry and find that only a few of the selected choices were added to the text field by the step.
- Add code snippet: `add_filter( 'gravityflowformconnector_update_field_values_form', array( gp_populate_anything(), 'hydrate_form' ), 10, 2 );`
- Repeat test with this branch and find all the selected choices are added.
